napari/_qt/menus/_tests/test_file_menu.py

```diff
@@ -43,6 +43,46 @@ def _(path):
     mock_read.assert_called_once()
 
 
+def test_plugin_display_name_use_for_multiple_samples(
+    make_napari_viewer, tmp_plugin: DynamicPlugin
+):
```

Comment:
did this test pass in an environment that contains any plugin that provides sample data?

Reply:
Individually yes, when running the whole suite I got IndexErrors. Trying a different approach now: using builtins.